api/api.yaml

```diff
@@ -536,6 +537,29 @@ paths:
         204:
           description: Notification processed successfully.
 
+  /internal/customer/{customerID}/task/{taskID}:
+    parameters:
+      - name: customerID
+        in: path
+        description: ID of the customer.
+        required: true
+        schema:
+          type: integer
+      - name: taskID
+        in: path
+        description: ID of the task.
+        required: true
+        schema:
+          type: string
+    put:
+      description: >
+        This endpoint acts as a proxy for updating the Task resource. Internal checks have to be done first before updating.
```

Comment:
Missing body?

Reply:
not defining a body acts the same as an object type body. This is also done for other services.